proto/cacheclient.proto

```diff
@@ -1025,3 +1028,35 @@ message _SortedSetLengthByScoreResponse {
     _Missing missing = 2;
   }
 }
+
+message _SortedSetUnionStoreRequest {
+  enum AggregateFunction {
+    SUM = 0;
+    MIN = 1;
+    MAX = 2;
+  }
+
+  message _Source {
+    bytes set_name = 1;
+    optional float weight = 2;
```

Comment:
I think this should not be optional

Reply:
Ok, I'll remove it here and let the SDK handle defaults.

Reply:
https://github.com/momentohq/client-protos/pull/310/commits/4d16a5dd8ce4a0cd9e6836374dd600d26d14442c